searchobject/src/main/java/org/jalau/at18/searchobject/controller/service/NotifyService.java

```diff
@@ -1,15 +1,29 @@
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
 package org.jalau.at18.searchobject.controller.service;
 
+import org.jalau.at18.searchobject.common.exception.NotifierTypeException;
 import org.jalau.at18.searchobject.model.objectrecognizer.recognizertypes.MatchInfo;
 import org.jalau.at18.searchobject.notify.Notifier;
 import org.jalau.at18.searchobject.notify.VerifyNotifier;
 import org.springframework.stereotype.Service;
 
 import java.util.List;
 
+/**
+ *
+ *
+ * @throws NotifierTypeExeption if the notifier type is not one of the available notifiers
```

Comment:
you probably meant `NotifierTypeException` referring to the exception on line 26.

Reply:
It's already changed